apiv2/db_import/importer.py

```diff
@@ -254,6 +258,13 @@ def load_func(
                 if tiltseries:
                     tiltseries_obj = tiltseries.import_to_db()
                     tiltseries_cleaner.mark_as_active(tiltseries_obj)
+                    parents = {"run": run_obj, "tiltseries": tiltseries_obj}
```

Comment:
nit: Do we need the run object here or could that be referenced from the tiltseries object?